src/actions/API.actions.js

```diff
@@ -95,6 +96,8 @@ export function createSUSIMessage(createdMessage, currentThreadID, voice) {
       Actions.sendToHardwareDevice(response);
 
       receivedMessage.text = response.answers[0].actions[0].expression;
+      // Setting Language received from User
+      receivedMessage.lang = response.answers[0].actions[0].language;
```

Comment:
> The action may still have no "language" object assigned, in that case still use the default language of the device.

Incase language attribute is not present like @Orbiter mentioned in https://github.com/fossasia/susi_android/issues/853, then it might throw an exception na?
But yea, like you said if its surely gonna be there then its redundant to add the check 👍